src/python/pants/option/global_options.py

```diff
@@ -201,3 +201,5 @@ def register_options(cls, register):
     register('--subproject-roots', type=list, advanced=True, fromfile=True, default=[],
              help='Paths that correspond with build roots for any subproject that this '
                   'project depends on.')
+    register('--force', type=bool, recursive=True,
```

Comment:
Maybe, but a surprising lie since `--cache-ignore` would invalidate local fps and local results, and then ... hit the cache if enabled and use its results. So decidedly not a cache ignore unless we tied the two together. That might be fine, its just that today the `invalidate` goal is not tied to the cache.